apps/opik-frontend/src/components/layout/SideBar/SideBar.tsx

```diff
@@ -70,32 +79,75 @@ const MAIN_MENU_ITEMS: MenuItem[] = [
   },
 ];
 
-const BOTTOM_MENU_ITEMS = [
-  {
-    path: buildDocsUrl(),
-    type: MENU_ITEM_TYPE.link,
-    icon: Book,
-    label: "Documentation",
-  },
-  {
-    path: "/$workspaceName/quickstart",
-    type: MENU_ITEM_TYPE.router,
-    icon: GraduationCap,
-    label: "Quickstart guide",
-  },
-];
-
 const HOME_PATH = "/$workspaceName/projects";
 
 type SideBarProps = {
   expanded: boolean;
   setExpanded: OnChangeFn<boolean | undefined>;
 };
 
+interface GetItemElement {
+  item: MenuItem;
+  content: React.ReactElement;
+  workspaceName: string;
+  linkClasses: string;
+  linkClickHandler: (event: React.MouseEvent<HTMLAnchorElement>) => void;
+}
+
+const getItemElementByType = ({
+  item,
+  content,
+  workspaceName,
+  linkClasses,
+  linkClickHandler,
+}: GetItemElement) => {
+  if (item.type === MENU_ITEM_TYPE.router) {
+    return (
+      <li key={item.id} className="flex">
+        <Link
+          to={item.path}
+          params={{ workspaceName }}
+          className={linkClasses}
+          onClick={linkClickHandler as never}
+        >
+          {content}
+        </Link>
+      </li>
+    );
+  }
+
+  if (item.type === MENU_ITEM_TYPE.link) {
+    return (
+      <li key={item.id} className="flex">
+        <a
+          href={item.path}
+          target="_blank"
+          rel="noreferrer"
+          className={linkClasses}
+        >
+          {content}
+        </a>
+      </li>
+    );
+  }
+
+  if (item.type === MENU_ITEM_TYPE.button) {
+    return (
+      <button key={item.id} onClick={item.onClick} className={linkClasses}>
```

Comment:
I believe there are some missing wrapper here, it should be inside a `li` as the other ones and inside the button probably (not semantically correct to add buttons inside an `ul`)

Also check the sidebar has 2 modes, collapsed and expanded, in the expanded view it looks off:
<img width="270" alt="Screenshot 2024-10-30 at 1 33 12 PM" src="https://github.com/user-attachments/assets/000f2d2c-3517-4244-a02b-69c4731fb382">